front/app/components/FormBuilder/utils.ts

```diff
@@ -25,6 +26,11 @@ export type FormBuilderConfig = {
   toolboxFieldsToExclude: ICustomFieldInputType[];
   formCustomFields: IFlatCustomField[] | undefined | Error;
 
+  isRequiredToggleDisabled?: (field: IFlatCustomFieldWithIndex) => boolean;
+  isResponseToggleDisabled?: (field: IFlatCustomFieldWithIndex) => boolean;
+  isEnableToggleDisabled?: (field: IFlatCustomFieldWithIndex) => boolean;
+  isTitleConfigurable?: (field: IFlatCustomFieldWithIndex) => boolean;
+
```

Comment:
Seemed like this should be controlled via the config, but I'm not sure if there is a better way?